lib-multisrc/mangathemesia/src/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesia.kt

```diff
@@ -165,23 +150,89 @@ abstract class MangaThemesia(
 
     override fun searchMangaNextPageSelector() = "div.pagination .next, div.hpage .r"
 
+    private fun selector(selector: String, contains: List<String>): String {
+        return contains.joinToString(", ") { selector.replace("%s", it) }
+    }
+
     // Manga details
     open val seriesDetailsSelector = "div.bigcontent, div.animefull, div.main-info, div.postbody"
-    open val seriesTitleSelector = "h1.entry-title"
-    open val seriesArtistSelector = ".infotable tr:contains(artist) td:last-child, .tsinfo .imptdt:contains(artist) i, .fmed b:contains(artist)+span, span:contains(artist)"
-    open val seriesAuthorSelector = ".infotable tr:contains(author) td:last-child, .tsinfo .imptdt:contains(author) i, .fmed b:contains(author)+span, span:contains(author)"
+
+    open val seriesTitleSelector = "h1.entry-title, .ts-breadcrumb li:last-child span"
+
+    private val artistLabels = listOf(
+        "artist",
+        "Artiste",
+        "Artista",
+        "الرسام",
+        "الناشر",
+        "İllüstratör",
+        "Çizer",
+    )
```

Comment:
These lists of labels aren't used anywhere else right? Then inline them to save memory.